daml.yaml

```diff
@@ -1,4 +1,4 @@
-sdk-version: 0.13.55
```

Comment:
You will also need to update this on the `.ci` so that circleci builds it with the intended sdk version as well